avatar badge component

som749898 opened the pull request:
avatar badge component completed
live preview: https://awesome-lalande-fd3882.netlify.app/
NOT RESPONSIVE

som749898 committed:
avatar badge component

avatar badge component completed

maitrakhatri reviewed (approved):
looks good to me

rohanmathur91 commented on CSS/color.css:
```diff
@@ -17,27 +16,27 @@
-  color: #666;
+  background-color: var(--warning-color);
+  color: var(--secondary-color);
 }
```
Use classes for different conditions, nth-child might cause a problem in styling in the future.

som749898 replied:
I have provided class name to boxes instead of using pseudo-class

rohanmathur91 replied:
Than its fine.

rohanmathur91 commented on CSS/main.css:
```diff
@@ -31,6 +31,12 @@ body {
 }
 
+#navbar {
+  padding: 20px 10px;
```
use rem if possible

som749898 replied:
converted pixel to rem

rohanmathur91 commented on CSS/main.css:
```diff
@@ -31,6 +31,12 @@ body {
+}
+
 #icon {
   font-size: 32px;
```
same as above.

som749898 replied:
converted pixel to rem

rohanmathur91 commented on CSS/main.css:
```diff
@@ -39,12 +45,6 @@ body {
   color: #fff;
 }
```
Have a variable for color

rohanmathur91 commented on CSS/main.css:
```diff
@@ -61,16 +61,24 @@ body {
   color: #fff;
 }
 
+.menu-list a:hover {
+  color: #276EF1;
```
same as above

som749898 replied:
added css variable

rohanmathur91 reviewed (commented):
Use variable for color, rest look good.

som749898 committed:
refactor: css variable and rem convertion

color were replaced with css variable and all pixel unit were converted to rem.  Naming of classes were corrected

rohanmathur91 reviewed (approved):
Looks goos to me.
